packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -21,6 +21,8 @@ const validManageableStates = new Set([
   undefined // not part of a package
 ]);
 
+export const CUSTOMOBJECTS_FULLNAME = 'CustomObject';
```

Comment:
Can we use `registry.types.customobject.name` from SDR (`import { registry } from '@salesforce/source-deploy-retrieve') instead ?

Reply:
Can you also please try to use `registry.types..` similarly for other types for the changes in the PR as well.

Reply:
No. By using `` registry.types.customobject.name from SDR (`import { registry } from '@salesforce/source-deploy-retrieve')``  I can get the type of the property of CustomObject like `string` but not the value of customobject.fullname .

Reply:
From the docs https://github.com/forcedotcom/source-deploy-retrieve/blob/main/HANDBOOK.md#the-registry-object and the example given there looks like `name` would be `fullname`. But we can double check with CLI team.

Reply:
Also please refer https://github.com/forcedotcom/salesforcedx-vscode/blob/e3ce5027f59a6ba735bcd78141b9efac66afa879/packages/salesforcedx-vscode-core/src/commands/util/parameterGatherers.ts#L195-L199

Reply:
From the conversation with CLI Looks like the full name and type name are different.